python/gps/algorithm/cost/cost_fk_blocktouch.py

```diff
@@ -0,0 +1,56 @@
+# TODO: This cost function is hard-coded and not easy to generalize.
```

Comment:
Is this true? If so, can you put inline comments in parts where it is hard-coded? (in case someone wants to use it for a different scenario)